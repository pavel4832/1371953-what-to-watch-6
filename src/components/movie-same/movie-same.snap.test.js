import React from 'react';
import {Provider} from 'react-redux';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import MovieSame from './movie-same';
import configureStore from 'redux-mock-store';
import MOCK_STORE from '../../mock/mock-store';

const mockStore = configureStore({});

it(`Should MovieSame render correctly`, () => {
  const store = mockStore(MOCK_STORE);
  const history = createMemoryHistory();
  const {container} = render(
      <Provider store={store}>
        <Router history={history}>
          <MovieSame />
        </Router>
      </Provider>
  );
  expect(container).toMatchSnapshot();
});
