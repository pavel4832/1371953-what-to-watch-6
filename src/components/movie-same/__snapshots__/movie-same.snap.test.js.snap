// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieSame render correctly 1`] = `
<div>
  <div
    class="page-content"
  >
    <section
      class="catalog catalog--like-this"
    >
      <h2
        class="catalog__title"
      >
        More like this
      </h2>
      <div
        class="catalog__movies-list"
      >
        <article
          class="small-movie-card catalog__movies-card"
        >
          <div
            class="small-movie-card__image"
          >
            This is mock VideoPlayer
          </div>
          <h3
            class="small-movie-card__title"
          >
            <a
              class="small-movie-card__link"
              href="/films/2"
            >
              Fantastic Beasts: The Crimes of Grindelwald
            </a>
          </h3>
        </article>
        <article
          class="small-movie-card catalog__movies-card"
        >
          <div
            class="small-movie-card__image"
          >
            This is mock VideoPlayer
          </div>
          <h3
            class="small-movie-card__title"
          >
            <a
              class="small-movie-card__link"
              href="/films/3"
            >
              Bohemian Rhapsody
            </a>
          </h3>
        </article>
        <article
          class="small-movie-card catalog__movies-card"
        >
          <div
            class="small-movie-card__image"
          >
            This is mock VideoPlayer
          </div>
          <h3
            class="small-movie-card__title"
          >
            <a
              class="small-movie-card__link"
              href="/films/4"
            >
              Macbeth
            </a>
          </h3>
        </article>
        <article
          class="small-movie-card catalog__movies-card"
        >
          <div
            class="small-movie-card__image"
          >
            This is mock VideoPlayer
          </div>
          <h3
            class="small-movie-card__title"
          >
            <a
              class="small-movie-card__link"
              href="/films/5"
            >
              Aviator
            </a>
          </h3>
        </article>
      </div>
    </section>
    <footer
      class="page-footer"
    >
      <div
        class="logo"
      >
        <a
          class="logo__link logo__link--light"
          href="/"
        >
          <span
            class="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            class="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            class="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        class="copyright"
      >
        <p>
          Â© 2019 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>
</div>
`;
